.prettierrc

```diff
@@ -0,0 +1,13 @@
+{
+  "trailingComma": "es5",
```

Comment:
have you tried parameter `all`? Is it change a lot of code while you reformat the code with this rule?